qtaf-xray-plugin/src/main/java/de/qytera/qtaf/xray/repository/xray/XrayTestRepositoryServer.java

```diff
@@ -0,0 +1,68 @@
+package de.qytera.qtaf.xray.repository.xray;
+
+import de.qytera.qtaf.core.QtafFactory;
+import de.qytera.qtaf.core.config.exception.MissingConfigurationValueException;
+import de.qytera.qtaf.core.gson.GsonFactory;
+import de.qytera.qtaf.http.RequestBuilder;
+import de.qytera.qtaf.http.WebService;
+import de.qytera.qtaf.xray.dto.response.steps.XrayServerTestStepResponseDto;
+import de.qytera.qtaf.xray.dto.response.steps.XrayTestStepResponseDto;
+import de.qytera.qtaf.xray.repository.jira.JiraIssueRepository;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Please add DocString

Reply:
Fixed in [8531207](https://github.com/Qytera-Gmbh/QTAF/pull/89/commits/85312073da996cc59b63375b5721a22ae2ebcfa7).